src/controllers/users.controller.js

```diff
@@ -0,0 +1,79 @@
+const usersServices = require('../services/users/users.services');
+
+const getUsers = (_, res) => {
+  return res.status(200).json(usersServices.getUsers());
+};
+
+const getUserById = (req, res) => {
+  const { id } = req.params;
+
+  const userId = Number(id);
+
+  if (!userId || isNaN(userId)) {
```

Comment:
The condition `!userId || isNaN(userId)` is redundant. Since `Number(id)` will always return a number, checking `isNaN(userId)` alone is sufficient to determine if the conversion was successful.